spec/services/bill_service_spec.rb

```diff
@@ -58,5 +58,55 @@
         expect(bill_data[:bill][:status]).to be_a(Integer)
       end
     end
+
+    describe "#rep_details" do
+      xit "returns rep details based on bill sponsors", :vcr do
+        sponsors = [{ name: "Abel Herrero"}]
+        full_state = "Texas"
+
+        rep_data = BillService.new({}).rep_details(full_state, sponsors)
+
```

Comment:
looks like these testss are still being worked on -- happy to pull down the branch and help if you'd like. I think it'd be best not to merge this to main until we have the tests sorted!